### ISSUE TITLE
Sql Cache In-Memory OLTP?

### ISSUE THREAD

**Issue by dauger** [aspnet/Caching#85]
Has having the option of using Sql Server 2014's in-memory OLTP instead of traditional tables been evaluated? It seems like it might be a good fit for a cache, but I'll admit that I have not done a deep dive on in-memory OLTP's capabilities & restrictions yet.

**Comment by glennc** [comment]
I don't think we have done any deep analysis on it. It's probably not a bad idea, but not something we have scheduled for our team at the moment.

**Comment by twenzel** [comment]
Is there any news on this topic?
Someone has already implemented a [prototype](https://www.nuget.org/packages/Microsoft.Extensions.Caching.Hekaton).
